src/components/VideoEditor.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from 'react';
+import '../index.css';
+import { Ambiance } from '../config/ambiance/types';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+import Stack from '@mui/material/Stack';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import { Channel } from '../config/ambiance/channels';
+
+
+type VideoEditorProps = {
+  documentId: string,
+  video?: Partial<Ambiance>,
+  videoUrlCode?: string,
```

Comment:
This value is already on the `video` object, lets remove this prop and just pull it off of `video.code`